src/Microsoft.PowerShell.ConsoleHost/host/msh/CommandLineParameterParser.cs

```diff
@@ -858,6 +858,19 @@ private bool ParseFile(string[] args, ref int i, bool noexitSeen)
             // of the script to evaluate. If -file comes before -command, it will
             // treat -command as an argument to the script...
 
+            bool? GetBoolValue(string arg)
+            {
+                if (arg.Equals("$true", StringComparison.OrdinalIgnoreCase))
```

Comment:
Ok, I think I understand what you're getting it by your comment below.  From a non-powershell shell like bash (which I thought would be the primary use case), I would think we want to support PowerShell syntax for passing $true and $false.  From within powershell, I suppose we shouldn't expect the user to escape $true to be just a string.  I can make this change.